docs/data/routes/docs/fundamentals/services/layout/jss-layout-api/en.md

```diff
@@ -0,0 +1,46 @@
+---
+name: jss-layout-api
+routeTemplate: ./data/component-templates/article.yml
+title: JSS Layout API
+---
+# JSS Layout API
+
+The JSS Layout API is a suite of services, clients and data fetchers that facilitate retrieving Sitecore layout data from Sitecore REST and/or GraphQL endpoints.
+
+## Invoking the REST Layout Service from JSS apps
+
+Sitecore JSS provides a simple API to make utilizing the Layout Service easier. Enter your configuration into the `fetchOptions` object and pass it into `dataApi.fetchRouteData()`. The `fetcher` option enables you to implement whichever data access method you wish. JSS ships with Axios, which can be imported from `src\dataFetcher.js`.
+
+The `dataApi` object is found in the `@sitecore-jss\sitecore-jss` package but is also exposed via the framework-specific SDKs.
+
+```javascript
+import { dataApi } from '@sitecore-jss/sitecore-jss';
```

Comment:
The `dataApi` object is obsolete and will be removed from the public API in a future release. All data fetching should be done by creating instances of services.

I think this whole page can be removed. The LayoutService, which you document in the next page is the correct way to fetch layout data.